local Config = require(game.ReplicatedStorage.Config)
local Remotes = require(game.ReplicatedStorage.Remotes)
local Base = require(game.ServerScriptService.Base)
local Spawner = require(game.ServerScriptService.Spawner)
local Pickup = require(game.ServerScriptService.Pickup)
local Placement = require(game.ServerScriptService.Placement)
local Income = require(game.ServerScriptService.Income)

Spawner.start()
Placement.setup()
Income.setup()
Income.start()

Pickup.onStash(function(player)
	Placement.refreshPrompts(player)
end)

Pickup.onEquipChanged(function(player)
	Placement.refreshPrompts(player)
end)